examples/treeview/treeview-1/js/tree.js

```diff
@@ -87,6 +88,14 @@ Tree.prototype.init = function () {
   this.firstTreeitem.domNode.tabIndex = 0;
 };
 
+Tree.prototype.setSelectedToItem = function (treeitem) {
+  if (this.selectedItem) {
```

Comment:
What do you think of mirroring how `setFocusToItem` is implemented with something like:

```js
  for (let i = 0; i < this.treeitems.length; i++) {
    const ti = this.treeitems[i];
    if (ti === treeitem) {
      ti.domNode.setAttribute('aria-selected', 'true');
    } else {
      ti.domNode.removeAttribute('aria-selected');
    }
  }
```